MatrixMultiplication/MatrixMultiplication/Matrix.cs

```diff
@@ -0,0 +1,106 @@
+namespace MatrixMultiplication;
+
+public class Matrix
+{
+    public readonly int[,] _matrix;
+    private static readonly Random Rand = new Random();
```

Comment:
```suggestion
    private static readonly Random Rand = new();
```
И всё, что private, пишется со строчной